server/datastore/mysql/migrations/tables/20241116233322_AddLuksDataToHostDiskEncryptionKeys.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241116233322, Down_20241116233322)
+}
+
+func Up_20241116233322(tx *sql.Tx) error {
+	_, err := tx.Exec(`ALTER TABLE host_disk_encryption_keys
+    		ADD COLUMN base64_encrypted_salt VARCHAR(255) NOT NULL DEFAULT '' AFTER base64_encrypted,
```

Comment:
What's the reason to use `AFTER`?

Reply:
Slight cleanliness thing when I (or others) use the `mysql` command line. Figured I might as well put encrypted/encryption related columns visually next to each other.